syntax = "proto3";

import "service/DataService.proto";
package apb.api.contentsstory;

option java_package = "jp.co.apb.server.api.proto.contentsstory";
option java_multiple_files = true;
option csharp_namespace = "Art.Framework.ApiNetwork.Grpc.Api.ContentsStory";

message RegisterPlayedRequest {
  int32 contents_story_id = 1;
}

message RegisterPlayedResponse {
  map<string, .apb.api.data.DiffData> diff_user_data = 99;
}

service ContentsStoryService {
  rpc RegisterPlayed(.apb.api.contentsstory.RegisterPlayedRequest) returns (.apb.api.contentsstory.RegisterPlayedResponse);
}

